pages/shardeum-liberty-alphanet/index.tsx

```diff
@@ -116,7 +116,7 @@ const AlphanetLanding: NextPage = () => {
 
       <FAQs
         heading={pageTranslation("page-alphanet-faq-h1")}
-        content={Array(7)
+        content={Array(6)
```

Comment:
here also, if there is a way to get the length of that json object, then this will become completely dynamic.

Reply:
Same with this as well, I am going to do that, it requires some restructuring.

Reply:
ok